Container Instances/template.json

```
@@ -0,0 +1,137 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "parameters": {
+    "aciGroupName": {
+      "type": "string"
+    },
+    "containerNamePrefix": {
+      "type": "string"
+    },
+    "agentCount": {
+      "type": "int",
+      "minValue": 1
+    },
+    "containerCoreCount": {
+      "type": "int",
+      "minValue": 1
+    },
+    "containerMemoryInGb": {
+      "type": "string"
```

```suggestion
      "type": "int"
```

Should it be type int https://docs.microsoft.com/en-us/azure/templates/microsoft.containerinstance/2018-10-01/containergroups#resourcerequests-object ?

---

It was set to string to support decimals